windows/windows-impl/src/main/java/com/duckduckgo/windows/impl/WindowsWaitlistFeatureDI.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2023 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.windows.impl
+
+import com.duckduckgo.anvil.annotations.ContributesRemoteFeature
+import com.duckduckgo.di.scopes.AppScope
+import com.duckduckgo.windows.api.WindowsWaitlistFeature
+
+@ContributesRemoteFeature(
+    scope = AppScope::class,
+    boundType = WindowsWaitlistFeature::class,
+    featureName = "windowsWaitlist",
+)
+@Suppress("unused")
+private interface WindowsWaitlistFeatureDI
```

Comment:
Should follow the naming that's already used in other places, i.e. `UnusedAutofillRemoteFeatureCodegenTrigger` for both the unused interface and the filename.